source/extensions/common/wasm/BUILD

```diff
@@ -23,6 +25,7 @@ envoy_cc_library(
         ":well_known_names",
         "//include/envoy/http:codes_interface",
         "//include/envoy/http:filter_interface",
+        "//include/envoy/server:wasm_interface",
```

Comment:
wasm_interface is a dependency required even neither v8 nor wavm is selected. Not a big deal

Reply:
Yeah, that's because we're still building WASM filters with the abstraction layer, even without runtimes. I expect it to say this way, since we're getting null sandbox, which will be always compiled in.